infra/test-buck2.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -xe
+
+# Run cpp example
+cd buck2/cpp
+../../buck2-exe build //:cpp_lib
```

Comment:
can't you create an alias instead and use an absolute path? Something like `alias buck2="$(realpath buck2-exe)"` and then just invoke buck2 normally

Reply:
done